src/Catalog/xslt/nuspec.xslt

```diff
@@ -290,39 +334,39 @@
   <xsl:template match="nuget:frameworkAssembly">
     <xsl:param name="path" />
 
-      <xsl:choose>
-        <xsl:when test="string-length(@targetFramework) &gt; 0">
-          <xsl:variable name="assemblyName" select="@assemblyName" />
-          <xsl:for-each select="obj:Split(@targetFramework)//item">
-            <ng:frameworkAssemblyGroup>
-              <rdf:Description>
-                <xsl:attribute name="rdf:about">
-                  <xsl:value-of select="obj:LowerCase(concat($base, $path, $extension, '#frameworkAssemblyGroup/', .))"/>
-                </xsl:attribute>
-                <ng:assembly>
-                  <xsl:value-of select="$assemblyName" />
-                </ng:assembly>
-                <ng:targetFramework>
-                  <xsl:value-of select="." />
-                </ng:targetFramework>
-              </rdf:Description>
-            </ng:frameworkAssemblyGroup>
-          </xsl:for-each>
-        </xsl:when>
-        <xsl:otherwise>
+    <xsl:choose>
```

Comment:
nit: Can we revert these indentation changes? makes the diff noisy

Reply:
I used VS auto format here. It's a bit noisy now, but it's at least internally consistent. Hopefully, we can just use autoformat for this doc any time we need to make changes going forward.